src/index.html

```diff
@@ -11,12 +11,110 @@
       content="ie=edge"
     />
     <title>Moyo header</title>
+    <link
+      rel="preconnect"
+      href="https://fonts.googleapis.com"
+    />
+    <link
+      rel="preconnect"
+      href="https://fonts.gstatic.com"
+    />
+    <link
+      href="https://fonts.googleapis.com/css2?family=Roboto:ital,wght@0,100;0,300;0,400;0,500;0,700;0,900;1,100;1,300;1,400;1,500;1,700;1,900&display=swap"
+      rel="stylesheet"
+    />
     <link
       rel="stylesheet"
       href="./style.css"
     />
   </head>
   <body>
-    <h1>Moyo header</h1>
+    <header class="header">
+      <a
+        href="index.html"
+        class="nav__logo"
+      >
+        <img
+          src="images/logo.png"
+          alt="Logo"
+        />
+      </a>
+      <nav class="nav">
+        <ul class="nav__list">
+          <li class="item">
+            <a
+              href="index.html"
+              class="is-active nav__link"
+            >
+              Apple
+            </a>
+          </li>
+
+          <li class="item">
+            <a
+              href="index.html"
```

Comment:
it will be better to use [anchors](https://www.htmlgoodies.com/html5/html5-navigation-using-an-anchor-tag-for-hypertext/)
```suggestion
              href="#samsung"
```